frontend/packages/knative-plugin/integration-tests/utils/install-serverless-operator.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName, retry } from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  servicesValidation,
+  revisionsValidation,
+  machineCount,
+  knativeServingNS,
+  pageSidebar,
+} from '../views/serverless.view';
+
+import {
+  switchPerspective,
+  Perspective,
+  sideHeader,
+} from '../../../dev-console/integration-tests/views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as catalogView from '../../../../integration-tests/views/catalog.view';
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+export const installKnOperator = async () => {
+  await crudView.isLoaded();
+  await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+  await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+  await crudView.isLoaded();
+  await catalogView.categoryTabsPresent();
+  await catalogView.categoryTabs.get(0).click();
+  await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+  await browser.wait(until.visibilityOf(installServerlessOperator));
+  await installServerlessOperator.element(by.linkText('Install')).click();
+  await operatorHubView.createSubscriptionFormLoaded();
+  expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+    'OpenShift Serverless Operator',
+  );
+  await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+  await operatorHubView.allNamespacesInstallMode.click();
+  expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+  expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+  await operatorHubView.createSubscriptionFormBtn.click();
+  await crudView.isLoaded();
+  await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+  await crudView.isLoaded();
+  await catalogPageView.clickFilterCheckbox('installState-installed');
+  expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(true);
```

Comment:
same as above. you can also break this into multiple utilities

Reply:
It will become a mess and multiple functions just because of those expect statements. And according to me expect functions are for validation, and doing validation is important rather than where we did.